test-cases/upgrades/manager-upgrade.yaml

```diff
@@ -19,7 +19,7 @@ space_node_threshold: 6442
 ip_ssh_connections: 'private'
 
 use_mgmt: true
-mgmt_port: 10090
+manager_prometheus_port: 12345
```

Comment:
this one is being used in `install_manager_agent`, you can update the code there to use `remote_scylla_manager_yaml` instead of using `sed` commands

Reply:
Done